code_challenges/code_wars/rotate_text.py

```diff
@@ -0,0 +1,7 @@
+def rotate(str_):
+    rotated_words = []
+    word = str_
```

Comment:
Why are you doing this line?

Reply:
i was thinking that i didn't want to change what i was looping over. originally i did enumerate(_str) and didn't want to change what i was looping over. but i changed it to loop over the range(len(_str)) which wouldnt be changing so i guess i dont need to do that now